components/Joblistings/JoblistingsContainer.tsx

```diff
@@ -106,6 +97,92 @@ type Props = {
 const isCurrentYear = (day: string): boolean =>
   dayjs(day).year() === dayjs().year();
 
+export const JoblistingItem = (props: {
+  node: JoblistingsContainer_joblisting$key;
+}): JSX.Element => {
+  const joblisting = useFragment(
+    graphql`
+      fragment JoblistingsContainer_joblisting on Joblisting {
+        slug
+        id
+        __typename
+        type
+        title
+        url
+        deadline
+        towns {
+          name
+        }
+        company {
+          name
+          logo(width: 800, height: 260)
+        }
+      }
+    `,
+    props.node
+  );
+
+  return (
+    <CompanyElement>
+      <Link
+        key={joblisting.id}
+        href={'/jobb/[slug]'}
+        as={`/jobb/${joblisting.slug}`}
+      >
+        <a>
+          <CompanyImage
+            src={joblisting.company.logo || '/static/itdagene-gray.png'}
+          />
+          <h3
+            style={{
+              fontWeight: 'normal',
+              fontSize: 20,
+              lineHeight: '24px',
+              color: 'black',
+              margin: '5px 0',
+              textAlign: 'center',
+            }}
+          >
+            {joblisting.title}
+          </h3>
+          <div style={{ color: 'gray', textAlign: 'center' }}>
```

Comment:
There's a lot of inline styling in this component, contrary to other components in the project

Reply:
hmm, just copied it over from JobListings :laughing:. But I think that's a bit out of scope of this PR :thinking: